scripts/install/mylar.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+# Brett 2021
+#shellcheck source=sources/functions/pyenv
+. /etc/swizzin/sources/functions/pyenv
+#shellcheck source=sources/functions/utils
+. /etc/swizzin/sources/functions/utils
+app_name="mylar"
+if [ -z "$MYLAR_OWNER" ]; then
+    if ! MYLAR_OWNER="$(swizdb get "$app_name/owner")"; then
+        MYLAR_OWNER="$(_get_master_username)"
+        echo_info "Setting ${app_name^} owner = $MYLAR_OWNER"
+        swizdb set "$app_name/owner" "$MYLAR_OWNER"
+    fi
+else
+    echo_info "Setting ${app_name^} owner = $MYLAR_OWNER"
+    swizdb set "$app_name/owner" "$MYLAR_OWNER"
+fi
+user="$MYLAR_OWNER"
+swiz_configdir="/home/$user/.config"
+app_configdir="$swiz_configdir/${app_name^}"
+app_configfile="$app_configdir/config.ini"
+app_group="$user"
+app_port="$(port 7000 11000)"
```

Comment:
Can we just pin this? Would be easier to update later if we need to see against it.